src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/RenamerProjectTreeActionHandler.cs

```diff
@@ -247,11 +242,15 @@ private async Task<bool> CheckUserConfirmationAsync(string oldFileName)
 
             string renamePromptMessage = string.Format(CultureInfo.CurrentCulture, VSResources.RenameSymbolPrompt, oldFileName);
 
-            bool userSelection = _userNotificationServices.Confirm(renamePromptMessage, out bool disablePromptMessage);
+            bool yesSelected = _userNotificationServices.Confirm(renamePromptMessage, out bool disablePromptMessage);
```

Comment:
Suggestion: `shouldRename` instead of `yesSelected`.

Reply:
done